src/assets/global.css

```diff
@@ -7,9 +7,12 @@
     --color-accent-dark: #cf9009;
     --color-accent2: #0eafaf;
 
+    --color-warning: #ecec0b;
```

Comment:
Over time, I should probably include more semantic color variables, like I used to do with Bootstrap. But for now, this is all I need.